spec/requests/api/generic_events_controller_create_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::GenericEventsController do
+  let(:response_json) { JSON.parse(response.body) }
+  let(:schema) { load_yaml_schema('post_event_responses.yaml', version: 'v2') }
+  let(:supplier) { create(:supplier) }
+  let(:access_token) { 'spoofed-token' }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+
+  let(:headers) do
+    {
+      'CONTENT_TYPE': content_type,
+      'Accept': 'application/vnd.api+json; version=2',
+      'Authorization' => "Bearer #{access_token}",
+    }
+  end
+
+  describe 'POST /generic_events' do
+    let(:event_attributes) do
+      {
+        occurred_at: Time.zone.now,
+        recorded_at: Time.zone.now,
```

Comment:
if it's important (I don't know) for them to be the same,
we should use
`let(:time_now) { Time.zone.now }`
but maybe that's not important

Reply:
No no, not important. Just need to be Time objects.